client/requestUtil.js

```diff
@@ -336,17 +334,21 @@ RequestUtil.prototype.shouldRetireOldSQSQueue = function (createdTimestamp) {
 
 /**
  * Checks do we need to use s3 list Object or SQS notifications
- * @param {number=} startAt return objects with timestamp >= startAt (e.g. 1482435340). Could be seconds or milliseconds
+ * @param {number=} timeToNormalize could be seconds or milliseconds
  * @param {number=} currentTime currentTime in milliseconds
- * @returns {number=}
+ * @returns {number=} the time for sure in milliseconds
 */
-RequestUtil.prototype.normalizeTimestampToMs = function (startAt, currentTime) {
-  let startAtToCheck = startAt
-  if (startAtToCheck && currentTime.toString().length - startAtToCheck.toString().length >= 3) {
-    startAtToCheck *= 1000
+RequestUtil.prototype.normalizeTimestampToMs = function (timeToNormalize, currentTime) {
+  if (!timeToNormalize) {
+    return 0
+  }
+
+  let timeToNormalizeToCheck = timeToNormalize
```

Comment:
why do we need this?

Reply:
in some tests `timeToNormalize` is not specified , so default value `undefined` is passed. I tried to avoid use of `undefined` in next lines and give exact 0 to the  code who invoked `normalizeTimestampToMs`